security/s2a/internal/crypter/aeadcrypter.go

```diff
@@ -1,16 +1,16 @@
 package crypter
 
-// S2AAeadCrypter is the interface for an AEAD cipher used by the S2A record
+// s2aAeadCrypter is the interface for an AEAD cipher used by the S2A record
 // protocol.
-type S2AAeadCrypter interface {
-	// Encrypt encrypts the plaintext and computes the tag of dst and plaintext.
+type s2aAeadCrypter interface {
+	// encrypt encrypts the plaintext and computes the tag of dst and plaintext.
```

Comment:
Having `encrypts encrypts...` looks a little strange style wise (and same for `decrypt` and `updateKey`). Is this demanded by the Go style guide. or can we change it to something cleaner?

Feel free to leave this comment unresolved until @cesarghali has a look.

Reply:
Documentation of unexported Go objects should start with the name of the object. The current documentation looks good to me.